src/component/calculatorShow.js

```diff
@@ -1,31 +1,23 @@
 import React from 'react';
 
-class CalculatorShow extends React.Component {
-  // eslint-disable-next-line no-useless-constructor
-  constructor(props) {
-    super(props);
-  }
-
-  showValue() {
-    const calculator = this.props.calculateResult;
+function CalculatorShow(props) {
```

Comment:
- [ ] Please use `arrow functions` instead of the `function` keyword, if you want to know more about why you should do this, take a look at the comment above 😄